### PR TITLE
Adds note about validating JSON

### PR DESCRIPTION
**Pull request by petems**
Quick one with validation example

### CODE REVIEW

File: content/source/docs/enterprise/private/automating-the-installer.html.md [hunk 1]
```diff
@@ -41,6 +41,15 @@ The settings file is JSON formatted. All values must be strings.  The example be
 }
 ```
 
+> Note: The JSON file must be valid JSON for the install to work, so it's best to validate it before using for an install.
+
+The easiest way to check the application config is valid JSON would be with `python`, which will be present on most Linux installs:
+
+```
+$ python3 -m json.tool settings.json
```
Comment (evanphx) [inline content/source/docs/enterprise/private/automating-the-installer.html.md]: Probably just go with `python` here.

File: content/source/docs/enterprise/private/automating-the-installer.html.md [hunk 2]
```diff
@@ -41,6 +41,15 @@ The settings file is JSON formatted. All values must be strings.  The example be
...
+Expecting property name enclosed in double quotes: line 8 column 5 (char 171)
```
Comment (evanphx) [inline content/source/docs/enterprise/private/automating-the-installer.html.md]: The example should show the output when it works probably not when there is an error. If you want to show an error output, show both.

File: content/source/docs/enterprise/private/automating-the-installer.html.md (hunk 1 above)
Comment (petems) [reply content/source/docs/enterprise/private/automating-the-installer.html.md]: Fixed 👍

File: content/source/docs/enterprise/private/automating-the-installer.html.md (hunk 2 above)
Comment (petems) [reply content/source/docs/enterprise/private/automating-the-installer.html.md]: Fixed 👍

**Comment by petems** [comment]
@evanphx Can I get a 👍 now I've made those changes?

**Pull request merged**